spring-observability-api/src/test/java/org/springframework/observability/ApiComponentsTest.java

```diff
@@ -69,6 +70,23 @@ void shouldRecordInstantEvent() {
 		assertThat(recording.getHighCardinalityName()).isEqualTo("test-instant-event-12345");
 		assertThat(recording.getTags()).containsExactly(Tag.of("testKey1", "testValue1", LOW),
 				Tag.of("testKey2", "testValue2", HIGH));
+		assertThat(recording.getWallTime()).isEqualTo(clock.wallTime());
+		assertThat(recording).hasToString(
```

Comment:
That's a very brittle test

Reply:
https://github.com/spring-projects/spring-observability/commit/087583c5d6923b5257a6e6b2ec0e430d0790655b